back/engines/commercial/analytics/app/models/analytics/fact_project_status.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: analytics_fact_project_statuses
+#
+#  project_id :uuid
+#  status     :string
+#  timestamp  :datetime
+#  date       :date
+#
+module Analytics
+  class FactProjectStatus < Analytics::ApplicationRecordView
+    belongs_to :dimension_date, foreign_key: :date
+    belongs_to :project
```

Comment:
Because this in the analytics module This needs to belong to dimension_project not project, although they're not much different, they are different types